### PR TITLE
 Remove unneeded artifacts from old test interface

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
I manually tested the HTML runners.

File: test-adapter/src/main/scala/org/scalajs/testadapter/HTMLRunnerBuilder.scala [hunk 1]
```diff
@@ -89,12 +88,11 @@ object HTMLRunnerBuilder {
       frameworkImplClassNames: List[List[String]],
       taskDefs: List[TaskDef]): String = {
 
-    val taskDefsString = Serializer.serialize(taskDefs)
+    def mkVar[T : Serializer](name: String, value: T) =
```
Comment (sjrd) [inline test-adapter/src/main/scala/org/scalajs/testadapter/HTMLRunnerBuilder.scala]: `T: Serializer` (I think it won't pass the Scalastyle checks)

**Comment by gzm0** [comment]
Note to self: Remove dependencies on json-simple.

**Comment by gzm0** [comment]
Updated.

**Pull request merged**